.rhcicd/stage-splunk-post-deployment-tests.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: notifications-splunk-post-deployment-tests
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdJobInvocation
+  metadata:
+    name: notifications-splunk-iqe-tests-${IMAGE_TAG}-${UID}
+  spec:
+    appName: notifications-backend
+    testing:
+      iqe:
+        debug: false
+        dynaconfEnvName: stage_post_deploy
+        filter: ''
+        marker: 'notif_engine and api'
```

Comment:
This was copied from https://github.com/RedHatInsights/notifications-backend/blob/master/.rhcicd/stage-engine-post-deployment-tests.yaml and requires changes.

Reply:
I'm not sure how this configuration helps identifying which IQE plugin should be used to run the tests...

Reply:
Let's try eventing-splunk-quarkus for the appName and 'splunk and smoke' for the marker